src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -621,19 +620,15 @@ private void testChangeFeedbackRecipient() {
 
         feedbackEditPage = getFeedbackEditPage();
 
-        assertTrue(feedbackEditPage.verifyPreviewLabelIsActive(1));
-        assertFalse(feedbackEditPage.verifyEditLabelIsActive(1));
         assertTrue(feedbackEditPage.verifyVisibilityPreviewIsDisplayed(1));
         assertFalse(feedbackEditPage.verifyVisibilityOptionsIsDisplayed(1));
 
         feedbackEditPage.clickQuestionEditForQuestion1();
-        feedbackEditPage.clickEditLabel(1);
+        feedbackEditPage.selectCustomVisibilityOption(1);
         feedbackEditPage.enableOtherFeedbackPathOptions(1);
         feedbackEditPage.selectRecipientToBe(FeedbackParticipantType.TEAMS, 1);
 
-        assertFalse(feedbackEditPage.verifyPreviewLabelIsActive(1));
-        assertTrue(feedbackEditPage.verifyEditLabelIsActive(1));
-        assertFalse(feedbackEditPage.verifyVisibilityPreviewIsDisplayed(1));
+        assertTrue(feedbackEditPage.verifyVisibilityPreviewIsDisplayed(1));
```

Comment:
Update logic of `verifyVisibilityPreviewIsDisplayed()` to return false when there is error